### ISSUE TITLE
Key Bindings are not working at all.

### ISSUE THREAD

**Issue by Antoshr** [OpenVR-Advanced-Settings/OpenVR-AdvancedSettings#523]
I currently having a problem with my binding as I'm trying to make myself be able to move in games like vrchat. It known as play space mover and for some strange reason it doesn't work for me. I have followed many guides and tutorials that take me to the same location and it just ends up not working at all. 
So the problem is that I try to change my key binding for advanced setting so I can move my avatar in vrchat up down left right when using full body. The current issue I'm getting is whatever I do to the binding nothing will work it will just use default binding from vrchat. Lets say I make my right hand drag to the A button (or use a community default that has A button set to that) all that will happen in vrchat is my avatar will jump and I will not be able to move my avatar up or down like I should. I have tried talking to people who use full body and they have not been able to help me, I have tried many things to try fix this like uninstalling everything, and reinstalling but nothing has seemed to work. 
I don't understand why it doesn't work for me even though I got the same setting as someone else and it works. I need a solution to make this functional because a play space mover is much required in full body to so I can calibrate the avatar well and move around so I'm not floating everywhere.

I would really like someone to help me with this, and would much appreciate any help you can offer.

 - OS: Windows 10
 - Advanced Settings Version: Newest (5.3.0

 - CPU: Ryzen 3700x
 - Graphics Card:  NVIDIA RTX 2070 SUPER

 - Oculus Rift S with 2 1.0 vive base stations

**Comment by ykeara** [comment]
Two pieces of advice.

1. assign tracker roles to be something besides "held in hand"
2. Make sure to turn on your Rift First before turning on your trackers.

combination of those two "should" fix it, as we only have limited control over how keybinds are handled this might be more of a SteamVR/OVR problem than us.

**Comment by Antoshr** [comment]
Hey thank you for the reply, I figured out the issue and it's all fixed now
and working fine. The issue is something to do with Steam vr playspace and
it's a really random issue that I figured out from just pressing stuff. And
I have already managed to get allot of random people binding working from a
simple fix. I'm going to address it just incase it's an unknown issue. But
basically the fix for it is to go to the advanced settings, navigate to
floor fix and then to press reset all changes made in this session and for
some reason it fixes the binding and allows for stuff like play space
moving. But yeah I don't know why this works but it did for me and many
other people that I bumped into.
